pkg/packaging/fetch.go

```diff
@@ -14,7 +16,8 @@ import (
 	"github.com/go-kit/kit/log/level"
 	"github.com/kolide/kit/fsutil"
 	"github.com/kolide/launcher/pkg/contexts/ctxlog"
-
+	"github.com/theupdateframework/go-tuf/client"
+	filejsonstore "github.com/theupdateframework/go-tuf/client/filejsonstore"
```

Comment:
```suggestion
 "github.com/theupdateframework/go-tuf/client/filejsonstore"
```

I think?

Reply:
It conflicts with the import of `"github.com/theupdateframework/go-tuf/client"` because that's actually the name of the package, which is a little annoying:

```
pkg/packaging/fetch.go:20:2: "github.com/theupdateframework/go-tuf/client/filejsonstore" imported as client and not used
```

Reply:
Ha. So they sloppily named the package _not_ what the folder is. oops!